examples/templates/tick-element-comparison.html

```diff
@@ -0,0 +1,92 @@
+---
+layout: default
+title: Forms / Tick elements
+category: _templates
+---
+<div class="p-strip is-shallow u-no-padding--top">
+  <div class="row">
+    <p>By default, checkbox and radio labels align with paragraphs.</p>
+  </div>
+  <div class="row">
+    <div class="col-3">
+      <input type="checkbox" id="checkExample1" checked>
+      <label for="checkExample1">Checkbox option 1</label>
+      <input type="checkbox" id="checkExample2">
+      <label for="checkExample2">Checkbox option 2</label>
+      <input type="radio" name="RadioOptions" id="Radio1" value="option1" checked>
+      <label for="Radio1">Radio option 1</label>
+      <input type="radio" name="RadioOptions" id="Radio2" value="option2">
+      <label for="Radio2">Radio option 2</label>
+    </div>
+    <div class="col-3">
+      <input type="radio" name="RadioOptions" id="Radio3" value="option3" >
+      <label for="Radio3">Radio option 3</label>
+      <input type="radio" name="RadioOptions" id="Radio4" value="option4">
+      <label for="Radio4">Radio option 4</label>
+      <input type="checkbox" id="checkExample3" checked>
+      <label for="checkExample3">Checkbox option 3</label>
+      <input type="checkbox" id="checkExample4">
+      <label for="checkExample4">Checkbox option 4</label>
+    </div>
+  </div>
+</div>
+<div class="p-strip is-shallow u-no-padding--top">
+  <div class="row">
+    <hr>
+    <p>In padded containers (table cells, some list items), checkboxes and labels need to align with text not wrapped in a tag. Use class "is-inline-label" on the label to achieve that.</p>
+  </div>
+</div>
+<div class="p-strip is-shallow u-no-padding--top">
+  <div class="row">
+    <table class="p-table--mobile-card">
+      <thead>
+        <tr>
+          <th>
+            <form>
+              <input type="checkbox" id="checkExample99" checked="">
+              <label for="checkExample99" class="is-inline-label">label</label>
+            </form>
+          </th>
+          <th>Power</th>
+          <th>Status</th>
+          <th>Owner | Pool</th>
+          <th class="u-align--right">Cores</th>
+          <th class="u-align--right">RAM(GB)</th>
+          <th class="u-align--right">Disks</th>
+          <th class="u-align--right">Storage(GB)</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <th aria-label="FQDN | MAC">
+            <form>
+              <input type="checkbox" id="checkExample1" checked="">
+              <label for="checkExample1" class="is-inline-label">label</label>
```

Comment:
There's a name conflict here - change the name of the checkbox

Reply:
@deadlight nice catch, thanks. fixed.